app/js/util/i18n.js

```diff
@@ -0,0 +1,33 @@
+import {LIBERAL, CONSERVATIVE, NDP, NO_POLICY_LISTED, TOPICS, MADE_BY, SUGGEST_EDIT} from './constants';
+
+let translations = new Map();
+translations.set(NDP, {en: 'NDP', fr: 'NPD'});
+translations.set(CONSERVATIVE, {en: 'Conservatives', fr: 'Conservateur'});
+translations.set(LIBERAL, {en: 'Liberals', fr: 'Libéral'});
+translations.set(NO_POLICY_LISTED, {en: 'No policy', fr: 'Absence de politique'});
+translations.set(TOPICS, {en: 'Topics', fr: 'Sujets'});
+translations.set(MADE_BY, {en: 'Made by', fr: 'Fait par'});
+translations.set(SUGGEST_EDIT, {en: 'Suggest edit', fr: 'Suggérer modifier'});
```

Comment:
Almost perfect!

I would only translate "Suggest edit" by "Suggérer une modification" in place :)